src/mesh/mesh_adaptation.cpp

```diff
@@ -48,15 +48,25 @@ void MeshAdaptation<dim,real,MeshType>::fixed_fraction_isotropic_refinement_and_
                                                                                         mesh_adaptation_param->h_coarsen_fraction);
     }
 
-    // Currently, the error indicator to flag p_adaptation is the same as h_adaptation, but it will likely change in future.
-    dealii::hp::Refinement::p_adaptivity_fixed_number(dg->dof_handler,
-                                                      cellwise_errors,
-                                                      mesh_adaptation_param->p_refine_fraction,
-                                                      mesh_adaptation_param->p_coarsen_fraction);
-    
-    // If a cell is flagged for both h and p adaptation, perform only p adaptation.
-    dealii::hp::Refinement::force_p_over_h(dg->dof_handler);
+//=========================================================================================================================================================
+    using MeshAdaptationTypeEnum = Parameters::MeshAdaptationParam::MeshAdaptationType;
+    MeshAdaptationTypeEnum mesh_adaptation_type = mesh_adaptation_param->mesh_adaptation_type;
+
 
+    if(mesh_adaptation_type == MeshAdaptationTypeEnum::h_adaptation){}
```

Comment:
Could you add a comment indicating why nothing is done if `h_adaptation` is selected?

Reply:
Fixed.